src/hotspot/share/gc/shared/plab.hpp

```diff
@@ -119,7 +119,7 @@ class PLAB: public CHeapObj<mtGC> {
   }
 
   // Sets the space of the buffer to be [buf, space+word_sz()).
-  void set_buf(HeapWord* buf, size_t new_word_sz) {
+  virtual void set_buf(HeapWord* buf, size_t new_word_sz) {
```

Comment:
I wonder if `override` is better.

Reply:
This is the base class so this is not an `override` but declaring the function as `virtual`.